src/host_toolchains.py

```diff
@@ -26,8 +26,8 @@
 SETUP_TOOLCHAIN = os.path.join(CR_BUILD_DIR, 'toolchain', 'win',
                                'setup_toolchain.py')
 V8_SRC_DIR = os.path.join(WORK_DIR, 'v8', 'v8')
```

Comment:
Looks like you can remove this line too.

Do you know why v8 would have been used to get these files in the first place?